source/_integrations/sunricher_dali_center.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: DALI Center
+description: Instructions on how to integrate DALI Center with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2025.10
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@niracler'
+ha_domain: sunricher_dali_center
+ha_platforms:
+  - light
+ha_integration_type: hub
+---
+
+The DALI Center integration connects Home Assistant to Sunricher SR-GW-EDA DALI gateways so you can manage DALI lighting from one place. Gateways are discovered on the local network and push updates keep entity state in sync.
+
+## Requirements
+
+- Sunricher SR-GW-EDA DALI gateway on the same network as Home Assistant
+- Commissioned DALI lighting devices connected to that gateway (tested with DALI DT6 dimmable drivers and DT8 tunable white/RGBW drivers)
+
+## Setup
+
+Configure the integration from the Home Assistant user interface; discovery finds supported gateways automatically.
+
```

Comment:
RE: discovery, in the future we could see if we could discover the device on the network via DHCP, mDNS or SSDP